index.html

```diff
@@ -5020,180 +5020,154 @@ <h3 id="playback-AudioBufferSourceNode">
-                            </li>
-                            <li>If <code>offset</code> &gt;=
-                            <code>actualLoopEnd</code> and
-                            <code>bufferTime</code> &lt;
-                            <code>actualLoopEnd</code>, set
-                            <code>enteredLoop</code> to <code>true</code> since
-                            playback began after the loop, and the playhead is
-                            now before the end of the loop.
-                            </li>
-                            <li>If <code>enteredLoop</code> is true, map the
-                            playhead position to the correct point within the
-                            loop body accounting for loop iterations:
-                              <ol>
-                                <li>While <code>bufferTime</code> &gt;= <code>
-                                  actualLoopEnd</code>, let
-                                  <code>bufferTime</code> equal
-                                  <code>bufferTime - (actualLoopEnd -
-                                  actualLoopStart)</code>.
-                                </li>
-                                <li>While <code>bufferTime</code> &lt;
-                                <code>actualLoopStart</code>, let
-                                <code>bufferTime</code> equal <code>bufferTime
-                                + (actualLoopEnd - actualLoopStart)</code>.
-                                </li>
-                              </ol>
-                            </li>
-                          </ol>
-                        </li>
-                        <li>If <code>bufferTime &gt;= 0</code> and
-                        <code>bufferTime &lt; buffer.duration</code>, emit the
-                        result of <code>playbackSignal(bufferTime)</code> as
-                        the output frame at <code>index</code>.
-                        </li>
-                        <li>Else, emit silence as the output frame at
-                        <code>index</code>.
-                        </li>
-                        <li>Set <code>bufferTime</code> to <code>bufferTime +
-                        (dt * computedPlaybackRate)</code>.
-                        </li>
-                      </ol>
-                    </li>
-                    <li>Increase <code>index</code> by 1.
-                    </li>
-                    <li>Increase <code>currentTime</code> by <code>dt</code>.
-                    </li>
-                  </ol>
-                </li>
-                <li>If <code>currentTime</code> &gt;= <code>stop</code>,
-                consider playback for the node to have ended. No further render
-                quanta for the node will occur.
-                </li>
-              </ol>
-            </li>
-          </ol>
+          <pre>
+// The AudioBuffer and context employed by this node
+var buffer;
```

Comment:
Style nit: I think the new ES6 style is to use "let" instead of "var" unless you really need "var". I don't think we do here.